gui/mozregui/build_runner.py

```diff
@@ -16,9 +16,11 @@ class GuiBuildDownloadManager(QObject, BuildDownloadManager):
     download_finished = Signal(object, str)
 
     def __init__(self, destdir, persist_limit, **kwargs):
-        QObject.__init__(self)
-        BuildDownloadManager.__init__(
-            self, destdir, session=get_http_session(), persist_limit=persist_limit, **kwargs
+        super().__init__(
+            destdir=destdir,
+            session=get_http_session(),
+            persist_limit=persist_limit,
+            **kwargs
```

Comment:
See: https://bugreports.qt.io/browse/PYSIDE-2304 and https://bugreports.qt.io/browse/PYSIDE-1564. For relevant release/dev notes: https://www.qt.io/blog/qt-for-python-6.5 and https://wiki.qt.io/Qt_for_Python_Development_Notes#20._April_2023.